data/org.gnome.software.gschema.xml

```diff
@@ -10,11 +10,6 @@
       <summary>Whether to manage updates in GNOME Software</summary>
       <description>If disabled, GNOME Software will hide the updates panel and not perform any automatic updates actions.</description>
     </key>
-    <key name="download-updates" type="b">
```

Comment:
Unless this is either also being removed upstream, or it's a downstream-only setting (which does not seem to be the case: https://gitlab.gnome.org/GNOME/gnome-software/blob/master/data/org.gnome.software.gschema.xml#L13), we should not remove it, but just make it always false and/or ignore its value when deciding what to do in the code.

Reply:
My 2 cents...  I don't think it makes sense to keep a gsetting, even if it is upstream, if the code does not actually implement what the gsetting says it should.

And, part of the complication here, is that while normally we could just keep the code and specify the default value that makes sense for Endless, in this case we previously exposed this setting via the UI, so some users may get into 3.4 with the gsetting true, and some with the gsetting false, and if the user had made any changes those changes would override whatever we set as the default, and the user will no longer have any UI to further adjust this value.

Reply:
@rshuler I agree with you. What I am suggesting is that we simply keep the setting in the schema, and patch the code to ignore its value.

Reply:
@cosimoc I had thought about keeping the setting in the schema but then thought that even if upstream starts using the setting elsewhere, we'd realize about it better if we don't have the setting at all. That said, I don't mind keeping it if for some reason it's better to keep it around.